src/nn/accumulator.h

```diff
@@ -17,7 +17,9 @@
 #ifndef ACCUMULATOR_H
 #define ACCUMULATOR_H
 
+#if defined(__x86_64__) || defined(_M_X64)

```

Comment:
I actually would prefer if we used the same `defined()` that are used below as that is really what's driving the include being added. I wouldn't mind it being copied into every block either that specifically requires it.